src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Log/Logger.cs

```diff
@@ -147,6 +147,60 @@ public static void Log(FunctionId functionId, LogMessage logMessage)
             }
         }
 
+        /// <summary>
+        /// log a specific event at the <see cref="LogLevel.Telemetry"/> severity level with a simple context message which should be very cheap to create.
+        /// </summary>
+        public static void LogTelemetry(FunctionId functionId, string? message = null)
```

Comment:
Do we also want overloads that take in https://github.com/dotnet/roslyn/blob/main/src/Workspaces/Core/Portable/Log/KeyValueLogMessage.cs and ensures the log level is at least high enough?